src/components/Reports/Reports.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import ReportsFilters from './ReportsFilters/ReportsFilters';
+import { getDomains, getPagesByDomainId } from './ReportsService';
+
+class Reports extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      domains: [],
+      domainSelected: null,
+      pages: [],
+      pageSelected: null,
+    };
+
+    this.changeDomain = this.changeDomain.bind(this);
+  }
+
+  async componentDidMount() {
+    const domains = await getDomains();
+    if (domains.length) {
+      const domainSelected = domains[0];
+      const pages = await getPagesByDomainId(domainSelected.id);
+      const pageSelected = pages.length ? pages[0] : null;
+      this.setState({
+        domains: domains,
+        domainSelected: domainSelected,
+        pages: pages,
+        pageSelected: pageSelected,
+      });
+    }
+  }
+
+  changeDomain = async (event) => {
+    const id = parseInt(event.target.value);
```

Comment:
Please, see [this comment](https://github.com/FromDoppler/doppler-webapp/pull/55/files#r264904246) in ReportsFilter.

Reply:
fixed